bash/sanity_checks.bash

```diff
@@ -55,6 +59,33 @@ function Perform_Internal_Sanity_Checks()
 
 #===================================================================================================
 
+function __static__Ensure_Valid_Module_Given()
+{
+    if [ "${HYBRID_module[Sampler]}" != "SMASH" ] \
+        && [ "${HYBRID_module[Sampler]}" != "FIST" ]; then
+        exit_code=${HYBRID_fatal_logic_error} Print_Fatal_And_Exit \
+            'The module specified for the Sampler run is not valid.' \
+            'Valid modules are: ' --emph 'SMASH' ' and ' --emph 'FIST' '.'
+    fi
+    return 0
+}
+
+function __static__Ensure_Additional_Paths_Given_For_Sampler()
```

Comment:
In this function I would use the `Ensure_Given_Files_Exist` utility function, to which you can pass text to be printed. E.g.
```bash
Ensure_Given_Files_Exist \
    'Some needed file for Thermal-FIST sampler was not specified in the configuration file' \
    -- "${HYBRID_fist_module[Particle_file]}" "${HYBRID_fist_module[Decays_file]}"
```